admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -49,13 +49,10 @@ const ApiKeysPage = () => {
 					method: "GET",
 				})
 			).json();
-			console.log("data: ", data);
 			setKeys(data);
 			setLoading(false);
 		} catch (err) {
-			toast.error(
-				"Check Backend Service for more info"
-			);
+			toast.error("Check Backend Service for more info");
```

Comment:
Can you give more information to the users please? `Check Backend Service for more info` is not acceptable